src/components/Timeline/index.tsx

```diff
@@ -20,6 +20,15 @@ const Timeline: React.FunctionComponent<TimelineProps> = ({
 }) => {
   const [expanded, setExpanded] = React.useState<boolean>(!!showAll);
 
+  const sorted = useMemo(
```

Comment:
I don't see why memoization is required here. Since Timeline will only re-render if `updates` changes, and since `sorted` does need to be recalculated every time that happens, then what is the purpose of the useMemo?

Reply:
I guess the component also re-renders when the `expanded` state changes, which does not affect the sort. So to avoid resorting whenever clicking the expand/collapse button.

But to be completely honest I just did it without thinking, because sorting is one of those things that I intuitively want to avoid doing more than is absolutely necessary.

So even if there wasn't anything forcing it to re-sort now, the memo makes sure that it stays that way when more functionality is added in the future (e.g. the notes, filtering, favorites and other upcoming timeline features).